azure_functions_worker/_thirdparty/aio_compat.py

```diff
@@ -13,7 +13,7 @@ def get_running_loop():
 
     This function is thread-specific.
     """
-    loop = asyncio._get_running_loop()
```

Comment:
This call was using the private method rather than the exposed method. Changed that to use the correctly exposed method.